docs/design-docs/rook-ceph/README.md

```diff
@@ -0,0 +1,213 @@
+    -rw-r--r--. 1 root root 44855 Jul 28 13:00 common.yaml
+    -rw-r--r--. 1 root root 31424 Jul 28 13:00 create-external-cluster-resources.py
+    -rw-r--r--. 1 root root 2641 Jul 28 13:00 create-external-cluster-resources.sh
+    drwxr-xr-x. 5 root root 47 Jul 28 13:00 csi
+    -rw-r--r--. 1 root root 363 Jul 28 13:00 dashboard-external-https.yaml
+    -rw-r--r--. 1 root root 362 Jul 28 13:00 dashboard-external-http.yaml
+    -rw-r--r--. 1 root root 839 Jul 28 13:00 dashboard-ingress-https.yaml
+    -rw-r--r--. 1 root root 365 Jul 28 13:00 dashboard-loadbalancer.yaml
+    -rw-r--r--. 1 root root 1554 Jul 28 13:00 direct-mount.yaml
+    -rw-r--r--. 1 root root 3308 Jul 28 13:00 filesystem-ec.yaml
+    -rw-r--r--. 1 root root 780 Jul 28 13:00 filesystem-test.yaml
+    -rw-r--r--. 1 root root 4286 Jul 28 13:00 filesystem.yaml
+    drwxr-xr-x. 2 root root 115 Jul 28 13:00 flex
+    -rw-r--r--. 1 root root 4530 Jul 28 13:00 import-external-cluster.sh
+    drwxr-xr-x. 2 root root 183 Jul 28 13:00 monitoring
+    -rw-r--r--. 1 root root 1409 Jul 28 13:00 nfs.yaml
+    -rw-r--r--. 1 root root 495 Jul 28 13:00 object-bucket-claim-delete.yaml
+    -rw-r--r--. 1 root root 495 Jul 28 13:00 object-bucket-claim-retain.yaml
+    -rw-r--r--. 1 root root 2306 Jul 28 13:00 object-ec.yaml
+    -rw-r--r--. 1 root root 2313 Jul 28 13:00 object-openshift.yaml
+    -rw-r--r--. 1 root root 698 Jul 28 13:00 object-test.yaml
+    -rw-r--r--. 1 root root 488 Jul 28 13:00 object-user.yaml
+    -rw-r--r--. 1 root root 3573 Jul 28 13:00 object.yaml
+    -rw-r--r--. 1 root root 19075 Jul 28 13:00 operator-openshift.yaml
+    -rw-r--r--. 1 root root 18199 Jul 28 13:00 operator.yaml
+    -rw-r--r--. 1 root root 1080 Jul 28 13:00 pool-ec.yaml
+    -rw-r--r--. 1 root root 508 Jul 28 13:00 pool-test.yaml
+    -rw-r--r--. 1 root root 1966 Jul 28 13:00 pool.yaml
+    -rw-r--r--. 1 root root 410 Jul 28 13:00 rgw-external.yaml
+    -rw-r--r--. 1 root root 2273 Jul 28 13:00 scc.yaml
+    -rw-r--r--. 1 root root 682 Jul 28 13:00 storageclass-bucket-delete.yaml
+    -rw-r--r--. 1 root root 810 Jul 28 13:00 storageclass-bucket-retain-external.yaml
+    -rw-r--r--. 1 root root 681 Jul 28 13:00 storageclass-bucket-retain.yaml
+    -rw-r--r--. 1 root root 1251 Jul 28 13:00 toolbox.yaml
+    -rw-r--r--. 1 root root 6089 Jul 28 13:00 upgrade-from-v1.2-apply.yaml
+    -rw-r--r--. 1 root root 14957 Jul 28 13:00 upgrade-from-v1.2-crds.yaml
+```
+
+After creating basic setup (`common.yaml`, `operator.yaml`, `cluster.yaml`) install toolbox (`toolbox.yaml`) as well for checking the ceph cluster status.
+
+**IMPORTANT**:
+
+ensure the **osd** container is created and running. It requires a storage device to be available on the nodes.
+
+During cluster startup it searches for the devices available and creates osd containers for them.
+
+Kubelet nodes have to use a default flag `enable-controller-attach-detach` set to `true`. Otherwise PVC will not attach to the pod.
+
```

Comment:
I could recommend to add here the location of the file where we can find the flag, it will help: 

/var/lib/kubelet/kubeadm-flags.env  on every worker nodes with kubelet. After that we need to restart kubelet: systemctl restart kubelet